client/my-sites/domains/domain-management/privacy-protection/privacy-protection-card/header.jsx

```diff
@@ -13,25 +12,13 @@ import { localize } from 'i18n-calypso';
  */
 import AddButton from './../card/add-button';
 
-const PrivacyProtectionCardHeader = ( {
-	displayCost,
-	selectedDomainName,
-	selectedSite,
-	translate,
-} ) => (
+const PrivacyProtectionCardHeader = ( { selectedDomainName, selectedSite, translate } ) => (
 	<header className="privacy-protection-card__header">
 		<h3>{ translate( 'Privacy Protection' ) }</h3>
 
 		<div className="privacy-protection-card__price">
 			<h4 className="privacy-protection-card__price-per-user">
-				{ translate( '{{strong}}%(cost)s{{/strong}} per domain / year', {
-					args: {
-						cost: displayCost,
-					},
-					components: {
-						strong: <strong />,
-					},
-				} ) }
+				{ translate( 'Free for all domains registered with us.' ) }
```

Comment:
Should this be "Free for all domains registered with WordPress.com"...not sure what the best way to word this is.